### PR TITLE
add db-demos product

### PR DESCRIPTION
**Pull request by ivy-tgr**

### CODE REVIEW

File: market/demos/db-demo/meta.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+	"id": "db-demos",
+	"name": "db Demos",
```
Comment (ivy-rew) [inline market/demos/db-demo/meta.json:3]: I'd use `Database Demos` in the market.

File: market/demos/db-demo/meta.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	"platformReview": "4.5",
+	"sourceUrl": "https://github.com/axonivy-market/db-demos",
+	"statusBadgeUrl": "https://github.com/axonivy-market/db-demos/actions/workflows/ci.yml/badge.svg",
+	"language": "English",
+	"industry": "Cross-Industry",
+	"tags": ["demo"],
+	"validate": true,
+	"mavenArtifacts": [
+		{
+			"name": "db Demos Product",
```
Comment (ivy-rew) [inline market/demos/db-demo/meta.json:15]: also here 'Database...'

**Review by ivy-rew — APPROVED** [review]
:100:

**Review by alexsuter — APPROVED** [review]
Nice 👍🏿

**Pull request merged**